Ginger/Ginger/BusinessFlowPages_New/NewAutomatePage.xaml.cs

```diff
@@ -652,6 +661,10 @@ private void UpdateUsedEnvironment(ProjEnvironment env)
 
         private void xGoToBFsTreeBtn_Click(object sender, RoutedEventArgs e)
         {
+            if(xAddActionMenuFrame != null && xAddActionMenuFrame.Content != null && xAddActionMenuFrame.Content.GetType() == typeof(MainAddActionsNavigationPage))
```

Comment:
please remove it from here, StopRecording should be done by RecordPage based on Context.PropertyChange event, it is ok to continue recording even if user shifted to another tab than Automate, but if he changed BF then we should stop

Reply:
this code is called from the top left arrow button which takes the control to businessflow list page, added the code to stop recording is  on Context.PropertyChange event.